From 196dc315ef4afa98fcecec11ad637aaa5b7fe323 Mon Sep 17 00:00:00 2001
From: Ulf Samuelsson <ulf@emagii.com>
Date: Wed, 19 Mar 2014 11:22:04 +0100
Subject: [PATCH 2/2] ltrace-0.7.3: use --with-sysroot

Signed-off-by: Ulf Samuelsson <ulf@emagii.com>
---
 scripts/build/debug/400-ltrace.sh |   10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/scripts/build/debug/400-ltrace.sh b/scripts/build/debug/400-ltrace.sh
index 44f8af2..f879373 100644
--- a/scripts/build/debug/400-ltrace.sh
+++ b/scripts/build/debug/400-ltrace.sh
@@ -28,7 +28,15 @@ do_debug_ltrace_build() {
 
     CT_DoLog EXTRA "Configuring ltrace"
     # ltrace-0.5.3, and later, don't use GNU Autotools configure script anymore
-    if [ "${CT_LTRACE_0_5_3_or_later}" = "y" ]; then
+    if [ "${CT_LTRACE_V_0_7_3}" = "y" ]; then
+        CT_DoExecLog CFG        \
+        ./configure             \
+            --build=${CT_BUILD} \
+            --host=${CT_TARGET} \
+            --prefix=/usr       \
+	    --with-sysroot=${CT_SYSROOT_DIR}
+    # ltrace-0.5.3, and later, don't use GNU Autotools configure script anymore
+    elif [ "${CT_LTRACE_0_5_3_or_later}" = "y" ]; then
         case "${CT_ARCH}:${CT_ARCH_BITNESS}" in
             x86:32)     ltrace_host="i386";;
             x86:64)     ltrace_host="x86_64";;
-- 
1.7.9.5

